README.md

```diff
@@ -117,6 +125,7 @@ winetricks win10 ie8 corefonts dotnet48 vcrun2019 dxvk
 ```
 cd "/home/$USER/.local/share/Duskhaven/"
 wget https://github.com/chtheiss/duskhaven_launcher/releases/latest/download/duskhaven_launcher.bin
+wget https://github.com/chtheiss/duskhaven_launcher/blob/main/images/Duskhaven.png
```

Comment:
Why do you need to do this?

Reply:
Ahh it's the ico. maybe rename it.

Reply:
to download the icon for the dekstop shortcut on linux
if you prefer it can be removed